Application/EdFi.Ods.AdminApi/Infrastructure/Security/SecurityExtensions.cs

```diff
@@ -26,6 +26,11 @@ IWebHostEnvironment webHostEnvironment
         var issuer = configuration.GetValue<string>("Authentication:IssuerUrl");
         var isDockerEnvironment = configuration.GetValue<bool>("EnableDockerEnvironment");
```

Comment:
@JBrenesSimpat what happened to my earlier edits to this file? I changed these lines to use a new extension method, `IConfiguration.Get`. That method does not return `null` - it performs the null check for you and throws an exception.